src/test/scala/uk/gov/hmrc/hmrcfrontend/views/helpers/hmrcStandardPageSpec.scala

```diff
@@ -67,26 +72,6 @@ class hmrcStandardPageSpec extends AnyWordSpecLike with Matchers with JsoupHelpe
   implicit val fakeRequest: FakeRequest[_] = FakeRequest("GET", "/foo")
   val requestWithNonce: FakeRequest[_]     = fakeRequest.withAttrs(TypedMap(RequestAttrKey.CSPNonce -> "a-nonce"))
 
-  private val messages = Map(
-    "cy" -> Map(
-      "back.text"                              -> "Back Welsh",
-      "govuk.exitThisPage.redirectUrlFallback" -> "www.some-safe-cy.url",
-      "govuk.exitThisPage.contentFallbackText" -> "Gadael y dudalen hon"
-    ),
-    "en" -> Map(
-      "back.text"                              -> "Back English",
-      "govuk.exitThisPage.redirectUrlFallback" -> "www.some-safe-en.url",
-      "govuk.exitThisPage.contentFallbackText" -> "Exit this page"
-    )
-  )
-
-  private def getMessages(lang: Lang = Lang("en")): Messages = {
-    val messagesApi = stubMessagesApi(messages, new DefaultLangs(Seq(lang)))
-    new MessagesRequest(FakeRequest(), messagesApi).messages
-  }
-
-  implicit val defaultMessages: Messages = getMessages()
-
```

Comment:
interesting that we had some tests checking for the fallback text of the message key and some with the messages set - makes sense to me to just use our messages - easy to update tests if messages change